src/content/protocols/aave.md

```diff
@@ -0,0 +1,888 @@
+
+The Aave DAO is Aave's onchain governance system, allowing `AAVE` holders, the lending protocol's governance token, to govern over various aspects ranging across treasury management, risk management and strategic initiatives.
+
+Different Aave protocol _Instances_ exist and are managed by the Aave DAO. These instances focus on specific use cases and chains. This review covers the Ethereum Mainnet Core instance.
+
+# Overview
+
+## Chain
+
+The report is concerned with the Aave V3 _Core_ instance deployed on Ethereum mainnet. Ethereum mainnet achieves a _Low_ centralization risk score.
+
+> Chain score: Low
+
+## Upgradeability
+
+The Aave v3 protocol can be organized in a number of logical modules or subsystems: _Core Lend & Borrow_, _Reserve Parameters_, _Treasury, Aave Ecosystem Reserves & Rewards_, _GHO Stablecoin_ and _Aave Governance_. Each of these subsystems exposes various degrees of control as explained in more detail below. Overall, these control vectors can result in the loss of user funds, loss of unclaimed yield or otherwise materially affect the expected performance of the protocol.
+
+### Core Lending & Borrowing
+
+This subsystem forms the core of Aave v3's borrow & lending features and keeps track of users' positions and related protocol state. It centers around the `Pool` contract that governs how debt in the system can be built up, collateralized and how positions can be liquidated. It includes Aave v3's `aTokens` and `variableDebtTokens`. All of these contracts are fully upgradeable through the _Aave Governance_ [subsystem](#aave-governance). An unintended (or malicious) contract upgrade can result in the loss of user funds, loss of unclaimed yield or otherwise materially affect the expected protocol performance.
+
+### Reserve Parameters
+
+The _Reserve Parameters_ subsystem is responsible for maintaining and updating critical risk parameters in the Aave v3 protocol. Among others, these include the set of enabled collateral and loan assets, their specific _liquidation loan-to-value_ ratios, _liquidation bonuses_. An upgradeable `PoolConfigurator` contract controls these parameters with permissions to implement changes delegated to different, specialized, multisig accounts such as the _Emergency Admin_ or _Risk Council_. A system of _Roles_, implemented using OpenZeppelin's ACL pattern, and _Stewards_ ensures that these multisig accounts can only implement changes in a limited range or over a certain time period. This system thus limits the risk of unintended updates through the multisig accounts. However, an upgrade of the `PoolConfigurator` contract could revert these safeguards thus reintroducing the risk of loss of funds, loss of unclaimed yield or a material change of the expected protocol performance.
```

Comment:
mention interest rate, caps ?

Reply:
suggestion wording:
over a certain time period -> in a certain frequency

Reply:
suggestion wording:
revert these safeguards -> remove these safeguards

Reply:
> However, an upgrade of the PoolConfigurator contract could revert these safeguards thus reintroducing the risk of loss of funds, loss of unclaimed yield or a material change of the expected protocol performance.

not sure if this is clear enough for a reader, but the new PoolConfigurator would also need to assign the permissions to multi-sigs directly, because the rate-limiter and magnitude-limiter sits in the steward contracts

Reply:
fixed